src/c++/perf_analyzer/genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -220,6 +220,8 @@ def _get_openai_input_text(self, req_inputs: dict) -> str:
             return payload["prompt"]
         elif self._response_format == ResponseFormat.OPENAI_VISION:
             content = payload["messages"][0]["content"]
+            if isinstance(content, str):
+                return content
```

Comment:
Variable number of images introduces a case, where no images are added to the prompt, so the content type is a string.
These two lines handles this case, since the next line in the file expects a content of a dictionary type.

Reply:
Ah I see. Thanks for explaining. Let's put a small comment for future reference :) 
```suggestion
            # When no images were included in the request input, the content
            # is same as text-only chat completions format (e.g. string).
            if isinstance(content, str):
                return content
```